### PR TITLE
feat(clickhouse): transpile oracle functions chr, lag, lead

### PR DESCRIPTION
**Pull request by sleshJdev**

### CODE REVIEW

File: sqlglot/dialects/clickhouse.py [hunk 1]
```diff
@@ -915,6 +915,22 @@ class Generator(generator.Generator):
...
+                "lagInFrame", expression.this, expression.args["offset"], expression.args["default"]
+            )
+
+        def lead_sql(self, expression: exp.Lead):
+            return self.func(
+                "leadInFrame",
+                expression.this,
+                expression.args["offset"],
+                expression.args["default"],
+            )
```
Comment (georgesittas) [inline sqlglot/dialects/clickhouse.py:932]: Let's move these to `TRANSFORMS` instead of defining methods.

File: tests/test_transpile.py [hunk 2]
```diff
@@ -21,6 +22,26 @@ class TestTranspile(unittest.TestCase):
     def validate(self, sql, target, **kwargs):
         self.assertEqual(transpile(sql, **kwargs)[0], target)
 
+    def test_functions(self):
```
Comment (georgesittas) [inline tests/test_transpile.py:25]: Move these to `test_clickhouse.py`, get rid of dialect imports and use string versions.

**Review by georgesittas — COMMENTED** [review]
Please share docs regarding these changes

File: tests/test_transpile.py (hunk 2 above)
Comment (sleshJdev) [reply tests/test_transpile.py:25]: But this test for transpiling. How to implement same test in there? Is it okay if I'll just use string-like dialect names?

File: sqlglot/dialects/clickhouse.py (hunk 1 above)
Comment (sleshJdev) [reply sqlglot/dialects/clickhouse.py]: Done 665b453e0f112369f85b04ea7705c0646977e89d

**Comment by georgesittas** [comment]
Thanks for the PR @sleshJdev, I'll get this in and do any final cleanups.

**Pull request merged**